client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -243,151 +188,149 @@ class TitanMailQuantitySelection extends React.Component {
 							},
 							components: {
-								strong: <strong />,
+								b: <strong />,
 							},
-							comment:
-								'%(price)s is a formatted price for each email mailbox, e.g. $3.50, €3.75, or PLN 4.50',
 						}
-					) }
-				</span>
-				<span>
+					) }{ ' ' }
+					{ purchaseCost.amount < renewalCost.amount
+						? translate(
+								'This is less than the regular price because you are only charged for the remainder of the current month.'
+						  )
+						: translate(
+								'This is more than the regular price because you are charged for the remainder of the current month plus any additional month until renewal.'
+						  ) }{ ' ' }
 					{ translate(
-						'All of your mailboxes are due to renew at {{strong}}%(price)s/mailbox/month{{/strong}} on %(date)s',
+						'All of your mailboxes are due to renew at the regular price of {{b}}%(fullPrice)s{{/b}} per mailbox when your subscription renews on {{b}}%(expiryDate)s{{/b}}.',
 						{
 							args: {
-								date: moment( expiryDate ).format( 'LL' ),
-								price: renewalCost.text,
+								proratedPrice: purchaseCost.text,
+								fullPrice: renewalCost.text,
+								expiryDate: moment( expiryDate ).format( 'LL' ),
 							},
 							components: {
-								strong: <strong />,
+								b: <strong />,
 							},
-							comment:
-								'%(price)s is a formatted price for each mailbox, e.g. $3.50, €3.75, or PLN 4.50; ' +
-								'%(date)s is a localized date of the rough form: 17 February 2021',
 						}
 					) }
-				</span>
-				<CompactCard className="titan-mail-quantity-selection__prorating-details">
-					<Gridicon icon="info-outline" size={ 18 } />
-					<span>{ prorationMessage }</span>
-				</CompactCard>
-			</React.Fragment>
+				</>
+			</Notice>
 		);
 	}
 
 	renderForm() {
-		const { isLoadingDomains, selectedDomainName, titanMonthlyProduct, translate } = this.props;
+		const { isLoadingDomains, titanMonthlyProduct, translate } = this.props;
 
 		if ( isLoadingDomains || ! titanMonthlyProduct ) {
 			return <AddEmailAddressesCardPlaceholder />;
 		}
 
 		return (
 			<>
-				<SectionHeader label={ translate( 'Choose Mailbox Quantity' ) } />
+				<SectionHeader label={ translate( 'Add New Mailboxes' ) } />
 
 				<Card>
-					<div className="titan-mail-quantity-selection__domain-info">
-						{ translate( "You're adding mailboxes for {{strong}}%(domainName)s{{/strong}}", {
-							args: {
-								domainName: selectedDomainName,
-							},
-							components: {
-								strong: <strong />,
-							},
-							comment: '%(domainName)s is a domain name',
-						} ) }
-					</div>
-					<div className="titan-mail-quantity-selection__mailbox-info">
-						{ this.renderCurrentMailboxCounts() }
-					</div>
-					<div>
-						<FormLabel>{ translate( 'Number of new mailboxes to add' ) }</FormLabel>
+					<div className="titan-mail-quantity-selection__form">
+						<FormLabel htmlFor="quantity">{ translate( 'Number Of Mailboxes' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Number of new mailboxes to add' )` *ES Score: 7*

Reply:
So there are two issues here:
1. I prefer to have labels be sentence case (i.e. first word only capitalised: `Number of mailboxes`), but it looks like we're not consistent across the UI (or possibly in EN-GB), so I don't feel too strongly about this.
2. If we don't shift to sentence case, then this needs to be `Number of Mailboxes` - "of" doesn't get capitalised in English. (And, no, I don't know why!)

Reply:
I wish there was a reference somewhere explaining what is the right way. I'll go with `Number of mailboxes`.